src/TodoList.js

```diff
@@ -5,40 +5,19 @@ import TodoListItem from './TodoListItem';
 // The todo arry
 ////////////////////////////
 
-const todoList = [
-     {
-       title: 'HelloReact',
-      id: 1,
-   }, {
-       title: 'Requirements',
-       id: 2,
-   },{
-     title: 'SettingupaReactProject',
-     id: 3,
-   },{
-     title: 'MeettheReactComponent.',
-     id: 4,
-   },{
-     title: 'React JSX',
-     id: 5,
-   },{
-     title: 'Lists in React',
-     id: 6,
-   },
-   
-   
-   ];
+
 
 ////////////////////////////////   
 // The todo function
 ///////////////////////////////
 
-function TodoList () {
+function TodoList ({todoList}) {
     return (
         <ul>
         {/*This map function will iterate through the array of object*/}
       
-      {todoList.map((todo) =><TodoListItem key={todo.id}  title={todo.title}  /> )} 
+      {/* {todoList.map((todo) =><TodoListItem key={todo.id}  title={todo.title}  /> )} */}
+      {todoList.map(todo =><TodoListItem key={todo.id} item = {todo}  /> )} 
```

Comment:
So this is totally fine, but a more semantic approach (like it was before) would be the following

```suggestion
      {todoList.map(todo =><TodoListItem key={todo.id} todo={todo}  /> )} 
```

Continued....